specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10773 @@
+      "externalDocs": {
+        "url": "https://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/pattern/PatternReplaceCharFilter.html"
+      }
+    },
+    "Similarity": {
+      "discriminator": "@odata.type",
+      "properties": {
+        "@odata.type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "@odata.type"
+      ],
+      "description": "Base type for similarity algorithms. Similarity algorithms are used to calculate scores that tie queries to documents. The higher the score, the more relevant the document is to that specific query. Those scores are used to rank the search results.",
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/azure/search/index-ranking-similarity"
+      }
+    },
+    "ClassicSimilarity": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.ClassicSimilarity",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Similarity"
+        }
+      ],
+      "description": "Legacy similarity algorithm which uses the Lucene TFIDFSimilarity implementation of TF-IDF. This variation of TF-IDF introduces static document length normalization as well as coordinating factors that penalize documents that only partially match the searched queries."
+    },
+    "BM25Similarity": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.BM25Similarity",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Similarity"
+        }
+      ],
+      "properties": {
+        "k1": {
+          "type": "number",
+          "format": "double",
+          "description": "This property controls the scaling function between the term frequency of each matching terms and the final relevance score of a document-query pair. By default, a value of 1.2 is used. A value of 0.0 means the score does not scale with an increase in term frequency.",
+          "x-nullable": true
+        },
+        "b": {
+          "type": "number",
+          "format": "double",
+          "description": "This property controls how the length of a document affects the relevance score. By default, a value of 0.75 is used. A value of 0.0 means no length normalization is applied, while a value of 1.0 means the score is fully normalized by the length of the document.",
+          "x-nullable": true
+        }
+      },
+      "description": "Ranking function based on the Okapi BM25 similarity algorithm. BM25 is a TF-IDF-like algorithm that includes length normalization (controlled by the 'b' parameter) as well as term frequency saturation (controlled by the 'k1' parameter)."
+    },
+    "VectorSearch": {
+      "type": "object",
+      "properties": {
+        "profiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchProfile"
+          },
+          "description": "Defines combinations of configurations to use with vector search."
+        },
+        "algorithms": {
```

Comment:
Just want to make sure this JSON property is `algorithms` as in `2023-07-01-Preview` this used `algorithmConfigurations`

Reply:
Thanks for double checking. We renamed it as part of the `2023-10-01-Preview` changes to `algorithms` since we wanted to stop appending "configurations" to everything